web/models/coherence/user.ex

```diff
@@ -21,6 +22,8 @@ defmodule CoursePlanner.User do
     field :comments, :string
     field :role, UserRole
     field :participation_type, ParticipationType
+    field :notified, Ecto.Date
+    field :notification_frequency_days, :integer
```

Comment:
I believe application level is enough, I'm putting also a select in the templates.